backend/tests/user/test_user_service.py

```
@@ -300,3 +300,98 @@ async def test_delete_user_invalid_object_id(mock_db_client, mock_get_database):
     # Expected result: False and never hit the DB
     mock_db_client.users.delete_one.assert_not_called()
     assert result is False
+
+
+# --- Tests for update_user_avatar_url ---
+
+
+@pytest.mark.asyncio
+async def test_update_user_avatar_url_success(mock_db_client, mock_get_database):
```

Ruff (0.12.2)

Unused function argument: `mock_get_database`

(ARG001)